ha_lineairdb.h

```diff
@@ -67,6 +67,29 @@ class LineairDB_share : public Handler_share {
   std::unique_ptr<LineairDB::Database> lineairdb_;
 };
 
+class Mysql_lineairdb_translator
+{
+public:
+  std::string nullFlags_;
+  std::string mysqlField_;
```

Comment:
Are there no problems with these two members being public?

A public member is manipulated from any point and time outside the class.
To restrict uncontrolled access, it is common to provide procedures with access methods such as `setter' and `getter', and to keep the members private.

I think `set_null_flags_in_buf` and `store_null_flags_to_mysql` would be similar to the setter method, right? 
If so, I recommend you change these two members to `private` and reconsider the access methods.

Reply:
I never thought of the risk of uncontrolled access and I was not aware of the reasons to use setter and getter.
I will make the member variables private and add getter for `mysqlField`.